Makefile

```diff
@@ -144,11 +146,11 @@ generate: controller-gen ## Generate code containing DeepCopy, DeepCopyInto, and
 	$(CONTROLLER_GEN) object:headerFile="hack/boilerplate.go.txt" paths="./..."
 
 .PHONY: fmt
-fmt: ## Run go fmt against code.
-	go fmt ./...
+fmt: goimports ## Run go goimports against code - goimports = go fmt + fixing imports.
+	$(GOIMPORTS) -w  ./main.go ./pkg ./api ./controllers
 
 .PHONY: vet
-vet: ## Run go vet against code.
+vet: ## Run go vet against code - report likely mistakes in packages.
 	go vet ./...
```

Comment:
ooooohhhh.... this is new! Well, for me at least 😎. `./...` used to also match the `vendor` dir, that's why we started listing our files/dirs manually. Great to see that it doesn't anymore. Well, since Go 1.9 already?! 🙈 Just  to be sure, `goimports` is still trying to modify files in `vendor`? 
https://go.dev/doc/go1.9#vendor-dotdotdot

Reply:
`goimports` v0.6.0 didn't  work with `./...`, so unless we add `./vendor` then goimports won't modify the files under vendor.